app/slackimport.go

```diff
@@ -109,6 +114,7 @@ func SlackParseChannels(data io.Reader, channelType string) ([]SlackChannel, err
 
 	var channels []SlackChannel
 	if err := decoder.Decode(&channels); err != nil {
+		// TODO: investigate
```

Comment:
Do we need this in the code? Maybe a ticket instead?

Reply:
This is a leftover from the session where I was investigating why some channels wouldn't import. Removed it.